mars/session.py

```diff
@@ -39,6 +39,24 @@ def run(self, *tensors, **kw):
             raise RuntimeError('Session has closed')
         return self._executor.execute_tensors(tensors, **kw)
 
+    def fetch(self, tensor):
+        from .tensor.expressions.datasource import TensorFetchChunk
```

Comment:
Check if the tensor is executed, if not raise an error.

Reply:
I think it should be checked in `Mars.Session.fetch`